pkg/kine/drivers/generic/generic.go

```diff
@@ -240,6 +242,9 @@ func Open(ctx context.Context, driverName, dataSourceName string, paramCharacter
 
 		FillSQL: q(`INSERT INTO kine(id, name, created, deleted, create_revision, prev_revision, lease, value, old_value)
 			VALUES(?, ?, ?, ?, ?, ?, ?, ?, ?)`, paramCharacter, numbered),
+
+		// TODO(MK-1397): Make policy and config configurable via CLI.
+		ac: NewAdmissionController(&LimitPolicy{MaxRunningTxn: 5000}),
```

Comment:
Maybe we should use 
```
ac: NewAdmissionController(&AllowAllPolicy{}),
```
here to disable this feature until https://warthogs.atlassian.net/browse/MK-1397 is implemented? I have not yet checked the typical amount of Txn when running microk8s (this is part of https://warthogs.atlassian.net/browse/MK-1398).